cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -108,9 +118,32 @@ def test_order_catalog_item(catalog_item, request):
     service_catalogs = ServiceCatalogs(catalog_item.name)
     service_catalogs.order()
     logger.info('Waiting for cfme provision request for service %s', catalog_item.name)
-    row_description = catalog_item.name
-    cells = {'Description': row_description}
+    cells = {'Description': catalog_item.name}
     row, __ = wait_for(requests.wait_for_request, [cells, True],
         fail_func=requests.reload, num_sec=1400, delay=20)
     assert 'Provisioned Successfully' in row.last_message.text
     DefaultView.set_default_view("Configuration Management Providers", "List View")
+
+
+@pytest.mark.tier(2)
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.6')
+@pytest.mark.ignore_stream("upstream")
+@pytest.mark.meta(blockers=[BZ(1420179, forced_streams=["5.6", "5.7", "upstream"])])
+def test_retire_ansible_service(catalog_item, request):
+    """Tests order catalog item
+    Metadata:
+        test_flag: provision
+    """
+    catalog_item.create()
+    service_catalogs = ServiceCatalogs(catalog_item.name)
+    service_catalogs.order()
+    logger.info('Waiting for cfme provision request for service %s', catalog_item.name)
+    cells = {'Description': catalog_item.name}
+    row, __ = wait_for(requests.wait_for_request, [cells, True],
+        fail_func=requests.reload, num_sec=1400, delay=20)
+    assert 'Provisioned Successfully' in row.last_message.text
+    myservice = MyService(catalog_item.name)
+    myservice.retire()
+    # this is to assert that service is deleted from VMDB after retirement
+    with error.expected(NoSuchElementException):
```

Comment:
**REQUIRED: ** I really was on the fence with putting this as an OPTIONAL but I really think we should solve this now. We shouldn't be importing anything from selenium in the test. If there is an error from deleting it we should either, make the delete return ```False```, this is something that we don't do across the board yet, OR catch the error inside the method and reraise it as another error that is inside our framework. I know it seems like a very picky request, but it is important to have the lines of delineation clear. Tests **shouldn't** ever need to talk to selenium, that's why we abstract things out. This means that one day if we were to replace selenium, we wouldn't have to go rooting through tests to fix it. Are we there everywhere yet? NO! but we really don't want to introduce more technical debt.